CONTRIBUTING.rst

```diff
@@ -380,37 +426,43 @@ checkout of MLflow:
       Generating ~/mlflow/mlflow/store/db_migrations/versions/b446d3984cfa_add_new_field_to_db.py
 
 
-These commands generate a new migration script (e.g. at
-``~/mlflow/mlflow/alembic/versions/12341123_add_new_field_to_db.py``) that you should then edit to add
-migration logic.
-
+These commands generate a new migration script (e.g., at ``~/mlflow/mlflow/alembic/versions/12341123_add_new_field_to_db.py``)
+that you should then edit to add migration logic.
 
-Launching the Development UI
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-We recommend `Running the Javascript Dev Server`_ - otherwise, the tracking frontend will request
-files in the ``mlflow/server/js/build`` directory, which is not checked into Git.
-Alternatively, you can generate the necessary files in ``mlflow/server/js/build`` as described in
-`Building a Distributable Artifact`_.
+Writing MLflow Examples
```

Comment:
I dont think we need to nest this heading under Python. Let me know what you think!

Reply:
Most of these examples are in Python, and @smurching saw it as fitting to put it under Python, at least for now.

Reply:
Oops, sorry - agree that it makes sense for examples to go into their own top-level section, i.e. "Developing and testing MLflow -> Writing MLflow examples", sorry for the confusion there.

Reply:
Looks like if a reader follows the "general prereqs" step, they'll actually install pytest & so be able to run the examples tests / develop examples

Reply:
Excellent! Will move it under "Developing and testing MLflow"